Islandora-Image-Segmentation/Islandora-Image-Segmentation-Solution-Pack
# Add segment list filter

## What is changed
Makes the segment list filter functional.

## How to test
1. Navigate to a page with segments. 
1. Select a segment type from the dropdown.
1. Verify that the numbers beside the categories match the number of segments of that type in the list.
1. Hit apply and verify that only the selected type of segment is being displayed.
